File: src/dataset/javascript/etl_exercism-javascript-bearguns/gemini/GeminiFlash_etl_v4.js
'use strict'
class ETL {
  transform(scores) {
    const newScores = {};

    for (const key in scores) {
      if (Object.hasOwn(scores, key)) {
        const score = Number(key);
        const letters = scores[key];

        for (let i = 0; i < letters.length; i++) {
          newScores[letters[i].toLowerCase()] = score;
        }
      }
    }

    return newScores;
  }
}

export default ETL;

File: src/dataset/cpp/etl_exercism-johnngugi/gemini/GeminiFlash_etl_v4.cpp
#include "etl.h"
#include <map>
#include <vector>
#include <algorithm>
#include <cctype>
using namespace std;

map<char, int> etl::transform(map<int, vector<char> > a)
{
    map<char, int> store;
    store.reserve(a.size());

    for (const auto& pair : a)
    {
        int score = pair.first;
        const vector<char>& letters = pair.second;

        for (char letter : letters)
        {
            store.emplace(tolower(letter), score);
        }
    }
    return store;
}